app/controllers/katello/concerns/api/v2/smart_proxies_controller_extensions.rb

```diff
@@ -0,0 +1,35 @@
+module Katello
+  module Concerns
+    module Api::V2::SmartProxiesControllerExtensions
+      extend ActiveSupport::Concern
+
+      included do
+        def_param_group :smart_proxy do
+          param :smart_proxy, Hash, :required => true, :action_aware => true do
+            param :name, String, :required => true
+            param :url, String, :required => true
+            param :download_policy, String, :required => false, :desc => N_('Download Policy of the capsule, must be one of %s') %
+                                      SmartProxy::DOWNLOAD_POLICIES.join(', ')
+            param_group :taxonomies, ::Api::V2::BaseController
+          end
+        end
```

Comment:
Note, i've filed http://projects.theforeman.org/issues/16812  to not have to do this anymore